lib/app/bloc_observer.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:bloc/bloc.dart';
+
+class SimpleBlocObserver extends BlocObserver {
+  @override
+  void onChange(BlocBase bloc, Change change) {
+    super.onChange(bloc, change);
+    print('${bloc.runtimeType} $change');
```

Comment:
minor, but `Change` only holds information about state changes, and not about events. Since cubits don't carry event information, this is fine, but when using `Bloc`, using `onTransition` can help since it holds that event information, and would help us to automate quite easily things like developer logs or analytics ;)

We can see during our huddle how to achieve this tomorrow

Reply:
awesome tip @jorgecoca 

Thanks about that!